apps/form-service/src/form/router/form.ts

```diff
@@ -377,6 +407,15 @@ export function createFormRouter({
     getForm(repository),
     accessForm(notificationService)
   );
+  router.get(
+    '/forms/:formId/submissions/:submissionId',
+    createValidationHandler(
+      param('formId').isUUID(),
+      query('code').optional().isString().isLength({ min: 1, max: 10 })
```

Comment:
should this be submissionId instead of code?

Reply:
Oh, I think those lines are me getting ahead of myself (different ticket) - I'm going to remove them